signer/src/keys.rs

```diff
@@ -254,7 +254,7 @@ impl sqlx::postgres::PgHasArrayType for PublicKey {
     }
 }
 
-#[cfg(feature = "testing")]
+#[cfg(any(test, feature = "testing"))]
```

Comment:
Do we ever run the tests without the `testing` feature? If that is so, I think we should do this update on all the other dummy implementations.

Reply:
> Do we ever run the tests without the `testing` feature? 

No we don't, and testing is one our default features too. 

I think I originally made more substantial changes to this file and rolled them all back and just didn't roll back this one. I don't mind doing the same for our other dummy implementations, I just would prefer to do it in a separate PR. I don't mind removing this from this PR too.

Reply:
Yeah sure, I think it makes sense to do it as a follow-up :)